### PR TITLE
Fixes Render.com integration

### LINKED ISSUE #43: Render.com issues
From render.com

> A render.yaml file was found, but there was an issue.
> 
> services[0].envVars[1]
>     must be from group or have a key and value
> services[0].envVars[2]
>     cannot simultaneously specify fields value and sync

### PR DESCRIPTION
**Pull request by robacarp**
fixes #43 

Also fixes the need to copy/paste a master key in by asking render.com to generate a value when the app is cloned.

### CODE REVIEW

File: render.yaml [hunk 1]
```diff
@@ -16,10 +22,9 @@ services:
         fromDatabase:
           name: hostedgpt
           property: connectionString
-      - key: RAILS_MASTER_KEY
+      - fromGroup: generated-secrets
```
Comment (anandvc) [inline render.yaml:25]: This is cool! Can you please also update the README.md file to remove that step of copy/pasting the key?

Comment (robacarp) [reply render.yaml:25]: Good call. I updated the instructions there a little. We should consider whether forking the repository is necessary for a render deploy -- if not, we can make the process even easier.

File: render.yaml [hunk 2]
```diff
@@ -4,6 +4,12 @@ databases:
     user: hostedgpt
     plan: free
 
+envVarGroups:
+- name: generated-secrets
+  envVars:
+    - key: rails_master_key
+      generateValue: true
+
```
Comment (krschacht) [inline render.yaml:12]: Oh, interesting! @robacarp how does this make it so we don’t have to copy that master.key value?

My assumption was: the master.key value is important and the fix would involve telling the rails app to find the key within the master.key file, not within the ENV. But it seems like you’ve fixed it somehow by telling render to generate a new one? Is there a discrepancy between this newly generated value and the value already stored in master.key?

**Comment by krschacht** [comment]
Lint is failing. :( We really need to remove this ruby linter. I have a GH Issue to go back to the super basic rubocop setup that I initially had. It only enforces a very small number (I think three) rules that I’ve found to actually be universally good. I hate enforcing optional stylistic differences or insignificant white space differences.

**Comment by al3rez** [comment]
> Lint is failing. :( We really need to remove this ruby linter. I have a GH Issue to go back to the super basic rubocop setup that I initially had. It only enforces a very small number (I think three) rules that I’ve found to actually be universally good. I hate enforcing optional stylistic differences or insignificant white space differences.

I already created a PR that reverting it back to previous setup.

**Comment by al3rez** [comment]
@robacarp please update your pr with the latest changes in main and lint issue should be resolved

**Pull request closed without merging**